src/main/java/com/ecommerce/Service/AuthenticationService.java

```diff
@@ -22,12 +21,12 @@ public class AuthenticationService {
 
     private final UserRepository userRepository;
     private final PasswordEncoder passwordEncoder;
-    private final JwtUtil jwtUtil;
+    private final JwtServiceImpl jwtServiceImpl;
     private final AuthenticationManager authenticationManager;
-@Builder
+    @Builder
     public AuthenticationResponse register(RegisterRequest request) {
         CustomUser user = CustomUser.builder()
-                .id(1L)
+                .userId(1L)
```

Comment:
this looks fishy. should the userId be `1` or dynamically changing?

Reply:
updated this, see commit c2b2de2